http/backup_service.go

```diff
@@ -106,6 +131,65 @@ func (h *BackupHandler) handleBackupShard(w http.ResponseWriter, r *http.Request
 	}
 }
 
+func (h *BackupHandler) handleBackupMetadata(w http.ResponseWriter, r *http.Request) {
+	span, r := tracing.ExtractFromHTTPRequest(r, "BackupHandler.handleBackupMetadata")
+	defer span.Finish()
+
+	ctx := r.Context()
+
+	baseName := time.Now().UTC().Format(influxdb.BackupFilenamePattern)
+
+	formWriter := multipart.NewWriter(w)
+	w.Header().Set("Content-Type", formWriter.FormDataContentType())
+
+	parts := []struct {
+		fieldname string
+		filename  string
+		writeFn   func(io.Writer) error
+	}{
+		{
+			"bolt",
```

Comment:
The API doc currently has this as [`boltdb`](https://github.com/influxdata/openapi/blob/master/src/oss/schemas/MetadataBackup.yml#L3), so we'll need to match it for codegen.

Higher-level Q: I'm wondering if we should use `kv` and `sql` as the part names instead of `bolt` and `sqlite`, to match the API endpoints & pseudo-abstract over the specific DB formats used. What do you think?

Reply:
I don't have a strong feeling, but having the parts named `kv` and `sql` seems like an improvement. I went ahead and updated the part names to `kv` and `sql`...it might make things a little more clear when parsing the response based on the endpoints, like you said.

Reply:
:+1: I've got a WIP branch to finish the openapi spec for the manifest & the restore endpoints, I'll update the part names as part of it.